### PR TITLE
minor touch-ups

### PR DESCRIPTION
**Pull request by daveyostcom**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/csharplang?pullRequest=4241) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/csharplang?pullRequest=4241) before we can accept your contribution.<br/><br/>:x: daveyostcom [sign now](https://cla.dotnetfoundation.org/dotnet/csharplang?pullRequest=4241)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/csharplang?pullRequest=4241) it.</sub>

File: proposals/discriminated-unions.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Discriminated unions / `enum class`
 
 `enum class`es are a new kind of type declaration, sometimes referred to as discriminated unions,
-where each every possible instance the type is listed, and each instance is non-overlapping.
+where every possible instance of the type is listed, and instances do not overlap.
```
Comment (333fred) [inline proposals/discriminated-unions.md:5]:
```suggestion
where every possible instance of the type is listed, and each instance is non-overlapping.
```

File: proposals/discriminated-unions.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ enum class Expr
 
 This is almost identical to the semantics of a top-level `enum class`, except that
 the nested enum class defines a nested root type, and everything below the nested enum
-class is a subtype of the nested root type, instead of the top-level root type.
+class is a subtype of the nested root type, rather than of the top-level root type.
```
Comment (333fred) [inline proposals/discriminated-unions.md:117]: Please revert this bit.

File: proposals/discriminated-unions.md [hunk 3]
```diff
@@ -157,21 +157,20 @@ on repeated invocations.
...
 
-There are some proposed adjustments to pattern matching and the switch expression to handle
+There are some proposed adjustments to pattern matching and the `switch` expression to handle
 `enum classes`. Switch expressions can already match types through the variable pattern, but
-for currently for reference types, no set of switch arms in the switch expression are considered
+for reference types, currently no set of switch arms in the switch expression are considered
 complete, except for matching against the static type of the argument, or a subtype.
 
-Switch expressions would be changed such that, if the root type of an `enum class` is the static
+Switch expressions would be changed such that if the root type of an `enum class` is the static
```
Comment (333fred) [inline proposals/discriminated-unions.md:165]: Please revert this one.

File: proposals/discriminated-unions.md [hunk 4]
```diff
@@ -157,21 +157,20 @@ on repeated invocations.
...
 type of the argument to the switch expression, and there is a set of patterns matching all
 members of the enum, then the switch will be considered exhaustive.
 
 Since value members are not constants and do not define new static types, they currently cannot
 be matched by pattern. To make this possible, a new pattern using the constant pattern syntax
-will be added to allow match against `enum class` value members. The match is defined to succeed
+will be added to allow matching against `enum class` value members. A match is defined to succeed
```
Comment (333fred) [inline proposals/discriminated-unions.md:171]:
```suggestion
will be added to allow matching against `enum class` value members. The match is defined to succeed
```

File: proposals/discriminated-unions.md [hunk 5]
```diff
@@ -157,21 +157,20 @@ on repeated invocations.
...
 if and only if the argument to the pattern match and the value returned by the `enum class` value
-member would be reference equal, although the implementation is not required to perform this
-check.
+member are reference equal, although the implementation is not required to perform this check.
```
Comment (333fred) [inline proposals/discriminated-unions.md:173]: Please revert this bit.

**Review by 333fred — CHANGES_REQUESTED** [review]
I'd prefer if some of these were left as is, as noted.

File: proposals/discriminated-unions.md (hunk 1 above)
Comment (daveyostcom) [reply proposals/discriminated-unions.md:5]: how about 'no two instances overlap'?

Comment (333fred) [reply proposals/discriminated-unions.md:5]: I prefer the original wording.

File: proposals/discriminated-unions.md (hunk 3 above)
Comment (daveyostcom) [reply proposals/discriminated-unions.md:165]: Ask some other writers what they think?

File: proposals/discriminated-unions.md (hunk 5 above)
Comment (daveyostcom) [reply proposals/discriminated-unions.md:173]: I think it's better if I just withdraw this, and you do what you want.

Comment (333fred) [reply proposals/discriminated-unions.md:173]: If you want to. There are still some fixes left we'd be happy to take.

**Comment by 333fred** [comment]
@daveyostcom I'm going to close this since there's been no updates to my feedback. Please ping if you would like to have it reopened.

**Pull request closed without merging**